src/current/cockroachcloud/stream-changefeed-to-snowflake-aws.md

```diff
@@ -34,7 +34,7 @@ If you have not done so already, [create a cluster]({% link cockroachcloud/creat
 
 ## Step 2. Connect to your cluster
 
-Refer to [Connect to a CockroachDB Dedicated cluster](https://cockroachlabs.com/docs/cockroachcloud/connect-to-your-cluster) for detailed instructions on how to to:
+Refer to [Connect to your cluster](https://cockroachlabs.com/docs/cockroachcloud/connect-to-your-cluster) for detailed instructions on how to to:
```

Comment:
This link will go to connecting to a [Standard Cluster](https://cloud-2-0--cockroachdb-docs.netlify.app/docs/cockroachcloud/connect-to-your-cluster). Is this ok?

Reply:
I also just noticed that there are 2 Step 2's on this page: Connect and Configure

Reply:
That is fine. Not sure what happened with the steps or when. Thanks for catching!